source/local/local.go

```diff
@@ -53,29 +100,25 @@ func (s Source) Discover() (source.Features, error) {
 	}
 
 	for _, file := range files {
-		hook := file.Name()
-		hookFeatures, err := runHook(hook)
+		fileName := file.Name()
+		hookFeatures, err := runHook(fileName)
```

Comment:
I would just use `lines` or `hookOutput` or smth, as that's what the data is at this point

Reply:
Done